Orange/widgets/data/owdatasets.py

```diff
@@ -283,6 +283,7 @@ def info(file_path):
             item1 = QStandardItem()
             item1.setData(" " if datainfo.islocal else "", Qt.DisplayRole)
             item1.setData(datainfo, Qt.UserRole)
+            item1.setData(QSize(26, 26), Qt.SizeHintRole)
```

Comment:
On my system this looks quite big.
24 still results in higher rows but looks better IMO.